frontend/apps/web/app/(mgmt)/[account]/transformers/components/SystemTransformersTable/columns.tsx

```diff
@@ -56,7 +53,7 @@ export function getSystemTransformerColumns(
               <div>
                 <NextLink
                   className="hover:underline"
-                  href={`/${account?.name}/transformers/systemTransformers/${transformer.source}`}
+                  href={`/${accountName}/transformers/systemTransformers/${transformerSource}`}
                 >
                   {row.original.name}
                 </NextLink>
```

Comment:
Ok so looking at this again, I realized that the only prop that needs to be passed in to the function is the account name. The transformer data is available on the `row`. So transformer source can be updated in the href to `{row.original.source}`.

Reply:
I did some changes @nickzelei

Reply:
Thanks! In the future you can let me know a PR is ready for re-review by clicking the cycle button in the reviewers section. Github will notify me you are ready for me to re-review this and it will show up in my queue!

<img width="320" alt="image" src="https://github.com/nucleuscloud/neosync/assets/2420177/8ef44089-e232-4d9f-80c9-6382b83a3437">

Reply:
Sure, thanks.
Learn a lot from this PR hope next time it would be better than this.